BigARTM_example_EN.ipynb

```diff
@@ -19,7 +19,7 @@
    "metadata": {},
    "source": [
     "This paper provides simple example of usage of new Python API. We will learn a model on the small dataset 'kos'.\n",
-    "Let's make necessary imports at first (note that you should have artm_model.py path in your Pyhton PATH variable):"
+    "Let's make necessary imports at first (note that you should have model.py path in your Pyhton PATH variable):"
```

Comment:
Not `model.py`, but entire `artm` module.
I think it is more correctly to write: _notice that you should have bigartm python api installed in your environment_